src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IThrowOperation.cs

```diff
@@ -0,0 +1,1918 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
```

Comment:
Consider moving the trait onto the class (probably needs a new class name for IOperation data flow tests).